core/src/main/java/org/apache/accumulo/fate/Fate.java

```diff
@@ -268,18 +268,18 @@ public long startTransaction() {
 
   // start work in the transaction.. it is safe to call this
   // multiple times for a transaction... but it will only seed once
-  public void seedTransaction(long tid, Repo<T> repo, boolean autoCleanUp) {
+  public void seedTransaction(long tid, Repo<T> repo, boolean autoCleanUp, String goalMessage) {
     store.reserve(tid);
     try {
       if (store.getStatus(tid) == TStatus.NEW) {
         if (store.top(tid) == null) {
           try {
+            log.info("FATE TxId: {} {}", tid, goalMessage);
```

Comment:
I would suggest using the following method for formatting all fate txids in logging and user utilities. This method was created because the code used to log fate txids in many different ways (hex and decimal for example).  Using this method they are logged consistently which is super important for grep/search of the logs for everything related to a particlar fate txid.

https://github.com/apache/accumulo/blob/4dc8d3effc9eeccd514317b3028bc10e1cf71c22/core/src/main/java/org/apache/accumulo/fate/FateTxId.java#L55

```suggestion
            log.info("Seeding {} goal: {}", formatTid(tid), goalMessage);
```